posthog/settings/data_stores.py

```diff
@@ -105,12 +105,7 @@
 KAFKA_HOSTS = ",".join(KAFKA_HOSTS_LIST)
 KAFKA_BASE64_KEYS = get_from_env("KAFKA_BASE64_KEYS", False, type_cast=str_to_bool)
 
-_primary_db = os.getenv("PRIMARY_DB", "postgres")
-PRIMARY_DB: AnalyticsDBMS
-try:
-    PRIMARY_DB = AnalyticsDBMS(_primary_db)
-except ValueError:
-    PRIMARY_DB = AnalyticsDBMS.POSTGRES
+PRIMARY_DB = AnalyticsDBMS.CLICKHOUSE
```

Comment:
What's the plan for this now that it's hardcoded and branches for the alternative value have been pruned all over?

Reply:
Start purging this too. I opted not to in this PR because it's a bit involved also. Will do as follow up